plenum/server/message_req_processor.py

```
@@ -173,12 +173,12 @@ def _validate_requested_preprepare(self, **kwargs):
                     # the input is expected as a dict (serialization with ujson==1.33)
                     pp = PrePrepare(**kwargs['pp'])
                     if pp.instId != kwargs['inst_id'] or pp.viewNo != kwargs['view_no']:
-                        logger.warning('{} found PREPREPARE {} not satisfying '
```

Let's not hide this exception. I think warning or error should be fine